exercises/concept/paolas-prestigious-pizza/.meta/Exemplar.elm

```diff
@@ -0,0 +1,92 @@
+module PaolasPrestigiousPizza exposing
+    ( Pizza
+    , ingredientsParser
+    , menuParser
+    , oneIngredientParser
+    , pizzaParser
+    , priceParser
+    , vegetarianParser
+    , wordParser
+    )
+
+import Parser exposing ((|.), (|=), Parser)
+
+
+type alias Pizza =
```

Comment:
Can we put this in a supporting file?

Reply:
I know we have done that for the last few concepts, but as a student, I really prefer seeing the types in the same place. It's idiomatic to have the types together with the code, and in this case the students would not be able to pass the tests if the modified it, I don't really see the benefit.

Reply:
I think there are pros and cons to both, but I do think we should be consistent. Can you and @mpizenberg argue it out, and then maybe we can create a ADR or add something to a readme or something.

Reply:
ah sorry I haven’t followed the discussion here so I might be off. Regarding file organization, except if we want to teach modules or opaque stuff, I think it’s better to have all in a single file for the student.

Reply:
I don't know if we need to be consistent, we only have 3 out of 13 exercises with support files. It made sense for `go` that had a complicated function that we wanted to hide from students, but here it's just one type.
Happy to discuss it though.

Reply:
Hi Jie / Jeremy, the Go exercise / Result concept was the one I was thinking of, see this comment from Erik:

https://github.com/exercism/elm/pull/438#discussion_r793701748

I'm happy for this to be merged as is.

I think it would be good to be consistent though, both within the Elm track, and potentially across the other exercism tracks. If there are occasionally things in the template file, students might think that it is ok to edit them for example, especially if they are sometimes moved to a non editable file. And it will give us something to review against.

Could we agree on a plan for this (maybe small things are included / above a certain size or complexity goes in a separate file) as a separate task? Maybe we should get an opinion from Erik as well?